client/my-sites/importer/importer-header.jsx

```diff
@@ -75,11 +75,11 @@ export const ImporterHeader = React.createClass( {
 		}
 
 		if ( includes( cancelStates, importerState ) ) {
-			return this.translate( 'Cancel', { context: 'verb' } );
+			return this.translate( 'Close', { context: 'verb, to Close a dialog' } );
 		}
 
 		if ( includes( stopStates, importerState ) ) {
-			return this.translate( 'Stop Import', { context: 'verb' } );
+			return this.translate( 'Importing...', { context: 'verb' } );
```

Comment:
Hi! I've found a possible matching string that has already been translated **71** times:
`translate( 'Importing...' )` *ES Score: 6.69*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).